docs/api/channels.rst

```diff
@@ -46,6 +48,208 @@ If you request for more detailed data, you will receive an object with the follo
 
     Read more about :doc:`Channels docs </components_and_bundles/components/Channel/models>`.
 
+Creating a Channel
+------------------
+
+To create a new channel you will need to call the ``/api/v1/channels/`` endpoint with the ``POST`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    POST /api/v1/channels/
+
++------------------------+----------------+------------------------------------------------------------------------+
+| Parameter              | Parameter type | Description                                                            |
++========================+================+========================================================================+
+| Authorization          | header         | Token received during authentication                                   |
++------------------------+----------------+------------------------------------------------------------------------+
+| code                   | request        | **(unique)** Channel identifier                                        |
++------------------------+----------------+------------------------------------------------------------------------+
+| name                   | request        | Name of the Channel                                                    |
++------------------------+----------------+------------------------------------------------------------------------+
+| taxCalculationStrategy | request        | Strategy which will be applied during processing orders in the channel |
++------------------------+----------------+------------------------------------------------------------------------+
+| defaultLocale          | request        | Locale code which is already saved in Locales menu                     |
```

Comment:
We are missing base currency in the definition table and a lot of other fields defined on model
```php
    /**
     * @var ZoneInterface
     */
    protected $defaultTaxZone;

    /**
     * @var string
     */
    protected $taxCalculationStrategy;

    /**
     * @var CurrencyInterface[]|Collection
     */
    protected $currencies;

    /**
     * @var LocaleInterface[]|Collection
     */
    protected $locales;

    /**
     * @var string
     */
    protected $themeName;

    /**
     * @var string
     */
    protected $contactEmail;

    /**
     * @var bool
     */
    protected $skippingShippingStepAllowed = false;

    /**
     * @var bool
     */
    protected $skippingPaymentStepAllowed = false;

    /**
     * @var bool
     */
    protected $accountVerificationRequired = true;
```

But it can be added in the next iteration